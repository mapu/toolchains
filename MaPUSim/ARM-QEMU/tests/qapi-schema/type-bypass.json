# Use of 'gen':false allows bypassing type system
{ 'command': 'unsafe', 'data': { 'arg': '**' }, 'returns': '**', 'gen': false }
