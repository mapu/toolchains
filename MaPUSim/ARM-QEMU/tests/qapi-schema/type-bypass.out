[OrderedDict([('command', 'unsafe'), ('data', OrderedDict([('arg', '**')])), ('returns', '**'), ('gen', False)])]
[]
[]
